package.yaml

```diff
@@ -1,5 +1,5 @@
 name:        yaml
-version:     0.10.4.0
+version:     0.10.4.1
```

Comment:
This is going to be a major version bump (0.11), since it breaks the existing API.

Reply:
Oh, for some reason in my head I hadn't considered that it was completely removing `Text.Libyaml` from the `yaml` API. My bad.

Reply:
Alternatively, we could have `yaml` re-export the `libyaml` module to avoid a major version bump.

Reply:
Eh, nevermind, the best practice would be for it to be re-exported under a different module name anyway.